### PR TITLE
Use 20240330

### PR DESCRIPTION
**Pull request by laozhoubuluo**
**Need add code to active postgres modules pgroonga to meet Firefish requirements from v20240319.**

Firefish now use a [rolling release cycle](https://en.wikipedia.org/wiki/Rolling_release) from v20240206.

### CODE REVIEW

File: deb/files/changelog [hunk 1]
```diff
@@ -1,3 +1,10 @@
+firefish (20240330-d202404010001+release) unstable; urgency=high
```
Comment (mizunashi-mana) [inline deb/files/changelog:1]: Please use `urgency=medium`
https://www.debian.org/doc/manuals/developers-reference/best-pkging-practices.html#selecting-the-upload-urgency

**Comment by mizunashi-mana** [comment]
@laozhoubuluo Thanks! But, v20240330 brings some failures on this package. Could you check?
* https://firefish.dev/firefish/firefish/-/blob/v20240330/Makefile is conflicted with `debian/rules`, please remove https://github.com/mizunashi-mana/firefish-dist-pkg/blob/main/deb/files/rules#L26 .
* Build flow is changed, could you fix?

**Comment by laozhoubuluo** [comment]
> workflow awaiting approval

I modified a version for test, but it looks like the newly pushed submission's workflow is awaiting approval.

**Comment by laozhoubuluo** [comment]
In order to compile the application, I need to insert the dev/config.env file, but the Debian packager and Debian specifications prevent me from doing so.
I tried to solve the problem by adding a patch, but if the compilation fails again, there may not be anything I can do to fix the problem.

**Comment by mizunashi-mana** [comment]
Thank you!

**Pull request merged**